app/models/concerns/hyku_addons/account_behavior.rb

```diff
@@ -68,14 +68,29 @@ def reset!
         DataCiteEndpoint.reset!
         Settings.switch!
         switch_host!(nil)
+        disable_tenant_cache
       end
 
       def switch_host!(cname)
         Rails.application.routes.default_url_options[:host] = cname
         Hyrax::Engine.routes.default_url_options[:host] = cname
       end
+
+      def setup_tenant_cache
+        return unless Flipflop.enabled?(:cache_enabled).presence
```

Comment:
Can you add a space beneath the guard statements to make them stand out from the rest of the business logic? Readability is second only to functionality in importance, so helping users understand what we are doing quickly is important to the project long term. 

Try to use `present?` and `blank?` for guard statments instead of `presence` as that has a different use case where a return value is required, not just a boolean. In this case neither is required as `Flipflop` does that for you.... it should be `return if Flipflop.enabled?(:cache_enabled)`.

So the whole thing would be:

```
      def setup_tenant_cache
        return unless Flipflop.enabled?(:cache_enabled)

        Rails.application.config.action_controller.perform_caching = true
        ActionController::Base.perform_caching = true
        Rails.application.config.cache_store = :redis_cache_store, { url: Redis.current.id }
        Rails.cache = ActiveSupport::Cache.lookup_store(Rails.application.config.cache_store)
      end
```